apps/api/pages/api/availability/_get.ts

```diff
@@ -119,8 +120,10 @@ const availabilitySchema = z
 async function handler(req: NextApiRequest) {
   const { prisma, isAdmin, userId: reqUserId } = req;
   const { username, userId, eventTypeId, dateTo, dateFrom, teamId } = availabilitySchema.parse(req.query);
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(req.headers.host ?? "");
   if (!teamId)
     return getUserAvailability({
+      orgSlug: isValidOrgDomain ? currentOrgDomain : undefined,
```

Comment:
We should be able to simplify by making sure currentOrgDomain is set only if isValidOrgDomain is true

Reply:
We could, we can tackle it as an improvement later on if you agree.

Reply:
Absolutely.